ios/FluentUI/Bottom Commanding/BottomCommandingController.swift

```diff
@@ -122,6 +129,27 @@ open class BottomCommandingController: UIViewController {
         }
     }
 
+    /// A layout guide that covers the on-screen portion of the current commanding view.
+    @objc public let layoutGuide = UILayoutGuide()
+
+    /// Height of the portion of the collapsed bottom chrome that's in the safe area.
+    ///
+    /// Valid after the root view is loaded.
+    ///
+    /// Use this to adjust `contentInsets` on your scroll views. This height won't change when the commanding UI is hidden or expanded.
+    @objc public var collapsedChromeHeight: CGFloat {
```

Comment:
couple of things: i don't think this variable name is super clear for other people what portion of the UI this is about for both iPad and iphone cases. Also, is there a need for clients that they need to know the collapsedchromeheight even in expanded cases? it can't be the "current content height" of the control?

Reply:
I used the word chrome to cover both the sheet and the bar. Do you have something in mind that would be more clear? 

As for the second point, in the sheet case, I specifically want this to only give the collapsed height, because the purpose of this property is to set scroll view bottom inset and it's not desirable for that to keep changing as the sheet expands. If the clients need to dynamically track where the bottom UI is right now, that's what the layout guide is for.

Reply:
Have we used the term "chrome" elsewhere? That doesn't feel like a very industry standard term. I like the idea of collapsedCommandingHeight

Reply:
collapseOffsetFromSafeArea? collapsedHeightFromSafeArea?

Reply:
Or collapsedHeightInSafeArea? This only has the part that's in the safe area.

Reply:
yup, I think that is slightly better even tho it is bit hard to explain for iPad case.

Reply:
Yeah that's a tough one as there's no concept of collapsing in the bottom bar. I think it's acceptable because the control client will know about the sheet representation and how it relates to that.

Reply:
I included more detail in the documentation to explain this will return the bottom bar height as well.